recipes/libs/cairo.yaml

```diff
@@ -0,0 +1,40 @@
+inherit: [autotools]
+
+metaEnvironment:
+    PKG_VERSION: "1.14.8"
```

Comment:
Any reason to use this five year old version? I mean, even if you'd like to stick in the 1.14 line, there's 1.14.12, which is *just* 3 years old...

Reply:
i'm stucked ;-)